### ISSUE TITLE
[NeuroLibre review] Wrong binder link

### ISSUE THREAD

**Issue by ltetrel** [Notebook-Factory/myelin-meta-analysis#5]
neurolibre/neurolibre-reviews#4

The binder link points to mybinder instead of binder.conp.cloud as stated in:
https://github.com/Notebook-Factory/myelin-meta-analysis/blob/main/content/_config.yml

Historical conversation here:
https://github.com/Notebook-Factory/myelin-meta-analysis/issues/3

**Comment by ltetrel** [comment]
@agahkarakuzu 
> To confirm that the book build returns the latest build, I will introduce some other changes. If I don't see those changes reflected, then we have other issues.

Indeed, if you don't see the introduced changed, then there is an issue with our backend. If you see them then the issue is on jupyter book side and we will need to see what to do.

**Comment by ltetrel** [comment]
To recapitulate:
1. It points to the wrong binderhub backend 
    `mybinder.org` instead of `binder.conp.cloud`
2. The repository target is not correct
    `neurolibre/myelin-meta-analysis` where it needs to be `Notebook-Factory/myelin-meta-analysis`
3. Path to notebook incorrect
    `tree/04/other_factors.ipynb` instead `lab/tree/content/04/other_factors.ipynb`

Overall we have this:
https://mybinder.org/v2/gh/neurolibre/myelin-meta-analysis/main?urlpath=tree/04/other_factors.ipynb
Instead of:
https://binder.conp.cloud/v2/gh/Notebook-Factory/myelin-meta-analysis/main?urlpath=lab/tree/content/04/other_factors.ipynb

**Comment by agahkarakuzu** [comment]
I fixed the repo target (2), but:

![image](https://user-images.githubusercontent.com/9632322/138172158-d2c5d4a7-74ad-437f-9ae1-9831158b90a2.png)

should not above deal with (1) and (3)?

**Comment by agahkarakuzu** [comment]
After all these, still wrong Binder link and jupyter interface:

http://neurolibre-data.conp.cloud/book-artifacts/Notebook-Factory/github.com/myelin-meta-analysis/3af04d90dbe36489a8fa913165f5c865d81c50e6/_build/html/

**Comment by ltetrel** [comment]
Yes, the link stays the same (without `lab` appended), that is why it is not using the jupyter lab interface...
Maybe @emdupre has any clue here ?

**Comment by emdupre** [comment]
What's the exact Jupyter Book command on the server side ?

**Comment by ltetrel** [comment]
`jupyter-book build --path-output book_dst_path_docker content`

**Comment by ltetrel** [comment]
@agahkarakuzu in all examples, I see that the url doe snot have a `/` at the end, maybe try with:
```
launch_buttons:
    binderhub_url: "https://binder.conp.cloud"
```

**Comment by ltetrel** [comment]
Latest commit did not help :'(
Proposing some changes

**Comment by ltetrel** [comment]
Ok the PR fixed the issue. The only thing remaining is the wrong path for the notebook files (3). @agahkarakuzu

**Comment by agahkarakuzu** [comment]
Do I need to change the following:

https://github.com/Notebook-Factory/myelin-meta-analysis/blob/57edf9b899aeb3ea67d28fe0769da83de666581b/content/_toc.yml#L6

to `content/01/selection` etc. 

Given that the `_toc` filed is in the `content` folder itself, would not it cause an error during the book build? If the `_toc` was at the root, I would see how this solution could work, but I guess we need something else here?

**Comment by agahkarakuzu** [comment]
@ltetrel do you have a comment for my latest comment?

**Comment by ltetrel** [comment]
I have no idea :(

**Comment by emdupre** [comment]
Should be resolved with #9  and #10 !

**Comment by agahkarakuzu** [comment]
Thank you @emdupre, closing the issue!
